docs/dictionary/command/revXMLRPC_SetMethod.lcdoc

```diff
@@ -2,7 +2,7 @@ Name: revXMLRPC_SetMethod
 
 Type: command
 
-Syntax: revXMLRPC_SetMethod <XML-RPC document>,<method name>
+Syntax: revXMLRPC_SetMethod <XML_RPC_documentID>,<method_name>
```

Comment:
Could you do these as camel case instead? It probably makes sense to keep that convention throughout the docs.